DEVELOPER_GUIDE.md

```diff
@@ -92,7 +92,15 @@ Next, go to the base directory (`cd ../..`) and run `yarn osd bootstrap` to inst
 
 From the base directory, run `yarn start`. This should start dashboard UI successfully. `Cmd+click` the url in the console output (It should look something like `http://0:5601/omf`). Once the page loads, you should be able to log in with user `admin` and password `admin`.
 
-## Integration Tests
+## Testing
+
+The OSSecurity Dashboards Plugin uses Jest, Cypress and Selenium and makes use of the [OpenSearch Dashboards Functional Test]( https://github.com/opensearch-project/opensearch-dashboards-functional-test) project.
```

Comment:
nit: Use the name of the repo here: `security-dashboards-plugin` instead of OSSecurity Dashboards Plugin